app/models/invoice.rb

```diff
@@ -6,4 +6,8 @@ class Invoice < ApplicationRecord
   belongs_to :customer
   belongs_to :merchant
   enum status: ['in progress', 'completed', 'cancelled']
+
+  def self.incomplete_invoices
+    Invoice.where(status: "in progress").order(created_at: :asc)
```

Comment:
you could also do `status: 0` here

Reply:
I actually like the "in progress" for clarity. Do you think it is a noticeable calculation?